src/v1/json/salutes.json

```
@@ -1,24 +1,56 @@
 {
   "salutes": [
+    "Stone.",
     "Rock on!",
+    "We rock!",
     "For Karl!",
+    "Rock solid!",
+    "For Teamwork!",
+    "By the Beard!",
+    "Rock and roll!",
+    "Rock... Solid!",
     "Rock and Stone!",
+    "Rock and Stone.",
+    "We\u0027re the best!",
+    "Rock and Stone!!",
+    "Galaxies finest!",
+    "ROCK! AND! STONE!",
+    "Rock and... Stone!",
     "For Rock and Stone!",
+    "We are unbreakable!",
     "ROCK... AND... STONE!",
+    "Let\u0027s Rock and Stone!",
+    "Rock me like a Stone!",
+    "Leave No Dwarf Behind!",
     "Rock and Stone forever!",
     "Rock and roll and stone!",
     "Rock and Stone everyone!",
     "Rock and Stone, Brother!",
     "Yeaahhh! Rock and Stone!",
+    "Rockitty Rock and Stone!",
     "None can stand before us!",
     "Like that! Rock and Stone!",
+    "Stone and Rock! No wait...",
     "Rock and Stone to the Bone!",
     "Yeah, yeah, Rock and Stone.",
+    "Gimmie a Rock... and Stone!",
     "We fight for Rock and Stone!",
     "Rock and Stone in the Heart!",
     "Did I hear a Rock and Stone?",
+    "Rock and Stone... Yeeaaahhh!",
+    "Come on guys! Rock and Stone!",
     "Stone and Rock! ...Oh, wait...",
-    "That's it lads! Rock and Stone!",
-    "If you don't Rock and Stone, you ain't comin' home!"
+    "That\u0027s it lads! Rock and Stone!",
```

Remove the unnecessary Unicode
```suggestion
    "That's it lads! Rock and Stone!",
```